test/e2e/pruner_test.go

```diff
@@ -242,3 +251,90 @@ func TestPrunerPodCompletes(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func TestPrunerIgnoreInvalidImageReferences(t *testing.T) {
+	ctx := context.Background()
+	te := framework.SetupAvailableImageRegistry(t, nil)
+	defer framework.TeardownImageRegistry(te)
+
+	cr, err := te.Client().ImagePruners().Get(ctx, "cluster", metav1.GetOptions{})
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	origSpec := cr.Spec.DeepCopy()
+
+	suspend := false
+	cr.Spec.Suspend = &suspend
+	cr.Spec.Schedule = "* * * * *"
+	cr.Spec.IgnoreInvalidImageReferences = true
```

Comment:
do you have an inverse test?  (ensures the arg is not passed, or is passed w/ a value of false, when this is not set?)

Reply:
now I have, fixed